upstream/CVE-2019-11254.json

```diff
@@ -0,0 +1,60 @@
+{
+  "id": "CVE-2019-11254",
+  "modified": "2020-03-26T18:55:26Z",
+  "published": "2020-03-26T18:55:26Z",
+  "summary": "kube-apiserver Denial of Service vulnerability from malicious YAML payloads",
+  "details": "The Kubernetes API Server component in versions 0-1.14, and versions prior to 1.15.10, 1.16.8 and 1.17.3 allows an authorized user who sends malicious YAML payloads to cause the kube-apiserver to consume excessive CPU cycles while parsing YAML.",
+  "affected": [
+    {
+      "package": {
+        "ecosystem": "kubernetes",
+        "name": "k8s.io/apiserver"
+      },
+      "severity": [
+        {
+          "type": "CVSS_V3",
+          "score": "CVSS:3.1/AV:N/AC:L/PR:L/UI:N/S:U/C:N/I:N/A:H"
+        }
+      ],
+      "ranges": [
+        {
+          "type": "SEMVER",
+          "events": [
+            {
+              "introduced": "0"
+            },
+            {
+              "fixed": "1.15.10"
+            },
+            {
+              "introduced": "1.16.0"
+            },
+            {
+              "fixed": "1.16.8"
```

Comment:
I'm confused. You said v1.16.7 has a fix, so shouldn't it be `1.16.7`?
https://github.com/aquasecurity/vuln-list-k8s/pull/74#discussion_r1343511322

Reply:
oh... right.
Thank you for seeing this. 
It was my typo.

Fixed in https://github.com/aquasecurity/vuln-list-k8s/pull/74/commits/4ed4d6d5b8d38a5513e714ed59dc5d9ff5656054